api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -62,13 +70,29 @@ trait ValidationErrorConverter {
       .getOrElse("")
   }
 
-  private def relevantFields(err: ValidationError): JsObject = {
+  private def relevantRow(err: ValidationError, vs: HmdaFileValidationState): HmdaFileRow = {
+    if (err.ts) vs.ts.get
+    else vs.lars.find(lar => lar.loan.id == err.errorId).get
+  }
+
+  private def relevantFields(err: ValidationError, row: HmdaFileRow): JsObject = {
     val fieldNames: Seq[String] = editDescriptions.find(e => e.editNumber == err.ruleName)
       .map(_.fieldNames).getOrElse(Seq())
 
-    val jsVals: Seq[(String, JsValue)] = fieldNames.map(n => (n, JsNumber(1)))
+    val jsVals: Seq[(String, JsValue)] = fieldNames.map { fieldName =>
+      val fieldValue = row.valueOf(fieldName)
+      (fieldName, jsonify(fieldValue))
+    }
 
     JsObject(jsVals: _*)
   }
 
+  private def jsonify(value: Any) = {
```

Comment:
Agreed. A more readable name is valuable. 

In 2e23a1de5637a61c4217e1216596406659ae88b0, I changed the name to `toJsonVal`, since it doesn't return actual JSON, just a `JsString` or `JsNumber`. Feedback/counter-offers welcome.